operations/services.md

```diff
@@ -1336,3 +1336,155 @@ Returns all companionships based on customers, reservations or reservation group
 | `CustomerId` | string | required | Unique identifier of [Customer](customers.md#customer). |
 | `ReservationId` | string | optional | Unique identifier of [Reservation](reservations.md#reservation). |
 | `ReservationGroupId` | string | required | Unique identifier of [Reservation group](reservations.md#reservation-group). |
+
+## Get all vouchers
+
+Returns all rate vouchers filtered by [Service](#service), voucher code or voucher identifier.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/vouchers/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ServiceIds": [
+        "bd26d8db-86da-4f96-9efc-e5a4654a4a94"
+    ],
+    "VoucherIds": [
+        "fe568bbd-1ecb-4bb2-bf77-96c3698de20d"
+    ],
+    "VoucherCodeValues": [
+        "TEST-VOUCHER-CODE"
+    ],
+    "Extent": {
+        "Vouchers": true,
+        "VoucherCodes": true,
+        "VoucherAssignments": true,
+        "Companies": false,
+        "Rates": false
+    },
+    "ActivityStates": [
+        "Active"
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceIds` | array of string | required | Unique identifiers of [Service](#service)s where the vouchers belong to. |
+| `VoucherIds` | array of string | optional | Unique identifiers of vouchers. |
+| `VoucherCodeValues` | array of string | optional | Value of voucher codes used by customers. |
+| `Extent` | [Voucher extent](#voucher-extent) | required | Extent of data to be returned. Whether only specific voucher info should be returned or related items as well. |
+| `ActivityStates` | array of string [Activity state](#activity-state) | required | Whether return only active, only deleted or both records. |
+
+#### Voucher extent
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Vouchers` | bool | optional | Whether the response should contain main information about vouchers. |
+| `VoucherCodes` | bool | optional | Whether the response should contain voucher codes used by customers. |
+| `VoucherAssignments` | bool | optional | Whether the response should contain assignments between vouchers and [Rate](#rate)s. |
+| `Companies` | bool | optional | Whether the response should contain detail of related companies. |
+| `Rates` | bool | optional | Whether the response should contain detail of assigned rates. |
+
+#### Activity state
+
+* `Active` - active records (the validity might be restricted by another parameter i.e. interval).
```

Comment:
this will be generic enum for all entities going forward, will the comment about validity interval apply to other entities?